### ISSUE TITLE
Several Issues (essentially all SIP-related features not working)

### ISSUE THREAD

**Issue by OfficialCRUGG** [koekeishiya/yabai#2283]
Hey there!

I have a variety of issues, mainly related to SIP-based features. I have just done the SIP setup steps once again, but still face all these issues.

Does anyone have any idea what could cause these issues?

yabai 7.1.1, macOS 14.5 (but same issues on macOS 14.1 earlier too)

## csrutil status Response

System Integrity Protection status: unknown (Custom Configuration).

Configuration:
	Apple Internal: disabled
	Kext Signing: enabled
	Filesystem Protections: disabled
	Debugging Restrictions: disabled
	DTrace Restrictions: enabled
	NVRAM Protections: disabled
	BaseSystem Verification: enabled
	Boot-arg Restrictions: disabled
	Kernel Integrity Protections: enabled
	Authenticated Root Requirement: enabled

This is an unsupported configuration, likely to break in the future and leave your machine in an unknown state.

## Issues

```
yabai -m config window_opacity on
yabai -m config active_window_opacity 1.0
yabai -m config normal_window_opacity 0.5

yabai -m config window_shadow off

yabai -m config window_animation_easing ease_in_out_sine
yabai -m config window_animation_duration 0.5
```
All windows are full opacity, have a shadow, and animations only appear very irregularly and when they do are glitched. Also when moving the window using a mouse action, it's extremely slow and delayed.

## Video

https://github.com/koekeishiya/yabai/assets/25248999/42adede0-a752-4ad0-89c6-573c84c34a1f

**Comment by aspauldingcode** [comment]
I get this sometimes too. Are you using a usb-video? Something like a usb hub with instantview?

**Comment by OfficialCRUGG** [comment]
I am using a DisplayLink Hub with the DisplayLink Driver. This is a must for me, because the M1 usually can only handle one display, but through DisplayLink, it works with three.

**Comment by OfficialCRUGG** [comment]
I don't think this is RAM related in my case. I always have plenty of ram left available.

**Comment by OfficialCRUGG** [comment]
"Found" the issue - `sudo yabai --load-sa` is in my .yabairc, but it apparently seems to not get triggered?! Everything works fine once I manually run the command.
Any ideas why this could be?

**Comment by 0xAquaWolf** [comment]
I was having a similar issue today when installing yabai from scratch today on stream, i followed this wiki link https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#configure-scripting-addition

and you have to make sure that your sudoers file has the last line commented out

it should read
`#includedir /private/etc/sudoers.d`

the hash tag is important, it begins with 2 but needs to only have one.

After i did this i had to restart my laptop

hope this helps
